src/core/router/transports/ssu/packet.cc

```diff
@@ -104,7 +104,7 @@ void SSUHeader::SetRekey(
 
 void SSUHeader::SetExtendedOptionsData(
     std::uint8_t* data,
-    std::size_t size) {
+    std::uint8_t size) {
```

Comment:
Would it be useful for `data` to be `const std::uint8_t* const` and `size` to be `const`?

Reply:
https://github.com/monero-project/kovri/issues/140#issuecomment-369775064.

Also, the const integral can be dealt with then too as it's not worth a rebase and re-trigger of the entire build system (we don't manipulate the data within the function).

Reply:
Understood, this seemed like a minor point to me as well (not worth the rebuild). Thanks for the clarification.